frontend/appflowy_flutter/lib/plugins/database_view/grid/presentation/widgets/header/type_option/number.dart

```diff
@@ -73,6 +73,7 @@ class NumberTypeOptionWidget extends TypeOptionWidget {
             height: GridSize.popoverItemHeight,
             alignment: Alignment.centerLeft,
             child: FlowyText.medium(
+              color: Colors.grey,
```

Comment:
Use or add a color that is based on `AFThemeExtension` or `Theme.of(context).colorScheme`.